via/db.py

```diff
@@ -0,0 +1,57 @@
+import alembic.command
+import alembic.config
+import zope.sqlalchemy
+from sqlalchemy import MetaData, create_engine, text
+from sqlalchemy.exc import ProgrammingError
+from sqlalchemy.orm import DeclarativeBase, MappedAsDataclass, Session
+
+
+class Base(MappedAsDataclass, DeclarativeBase):
```

Comment:
`MappedAsDataclass` enables SQLAlchemy 2's dataclasses support (so Via's SQLAlchemy model classes are dataclasses), which is really great